schemas/iso19139/src/main/plugin/iso19139/formatter/jsonld/iso19139-to-jsonld.xsl

```diff
@@ -249,7 +266,9 @@
         {
         "@type":"DataDownload",
         "contentUrl":"<xsl:value-of select="gmd:linkage/gmd:URL/text()"/>",
-        "encodingFormat":"<xsl:value-of select="gmd:protocol/gco:CharacterString/text()"/>"
+        "encodingFormat":"<xsl:value-of select="gmd:protocol/*/text() or gmd:protocol/*/@xlink:href"/>",
```

Comment:
This will return a boolean no ? 
Maybe replace by
```
p = normalize-space(gmd:protocol/*/text())
if($p != '') then $p else gmd:protocol/*/@xlink:href
```

Reply:
thanx for suggestion, added this to conversion